appengine/flexible/TwelveFactor/TwelveFactor.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>

```

Comment:
This is quite an old app. I know the VS templates are old but we should not creating new apps targetting .NET Core 1.0. I suggest you move to 2.0 to showcase this.